src/alfasim_sdk/alfasim_sdk_api/api.h

```diff
@@ -620,6 +620,24 @@ DLL_EXPORT int get_deposition_thickness(
    void* ctx, double** out, int phase_id, enum TimestepScope ts_scope, int* size
 );
 
+/*!
+    Sets a warning message at the solver logger.
+
+    @param[in] ctx ALFAsim's plugins context.
+    @param[in] warning_message Warning Message.
+    @return An #error_code value.
+*/
+DLL_EXPORT int set_warning_message(const char* warning_message);
+
+/*!
+    Sets an information message at the solver logger.
+
+    @param[in] ctx ALFAsim's plugins context.
+    @param[in] info_message Information Message.
+    @return An #error_code value.
+*/
+DLL_EXPORT int set_information_message(const char* info_message);
```

Comment:
`set_` seems to imply there is only one of each.
Maybe the prefix should be `add_` or `log_` (with a matching change to the documentation).

Also note that the signature is missing the `ctx` argument.

Reply:
I agree with both modifications. 
- [ ] change the prefix of the function names to `log_`
- [ ] add `ctx` parameter

Reply:
done :D